gui/src/hooks/config.ts

```diff
@@ -29,6 +30,7 @@ const initialConfig = {
   doneOnboarding: false,
   watchNewDevices: true,
   lang: 'en',
+  feedbackSound: true,
```

Comment:
```suggestion
  feedbackSound: false,
```
I personally would have any sound setting default to false. What do you think @loucass003 ?

Reply:
I would not.

Reply:
Why?

Reply:
Because it's an important UX feedback.